force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -73,6 +80,15 @@ export default class AccountModelSettings extends LightningElement {
         }
     }
 
+    @wire(getLeadConversionAccountNamingSettingsViewModel)
+    leadConversionAccountNamingSettingsViewModel({ error, data }) {
+        if (data) {
+            this.leadConversionAccountNamingSettingsViewModel = data;
+        } else if (error) {
+            //console.log("error retrieving leadConversionAccountNamingSettingsViewModel");
```

Comment:
I am not sure if this was intentional or not. I will leave it up to EDA reviewers to provide insight into EDA UI error handling.

Reply:
@MatthewBlanski just confirming this is OK with you, I was following the pattern of the other @wire's